rom/dev/src/rom.ld

```diff
@@ -26,6 +26,7 @@ FHT_ORG          = 0x50002000;
 LDEVID_TBS_ORG   = 0x50003000;
 FMCALIAS_TBS_ORG = 0x50003400;
 PCR_LOG_ORG      = 0x50003800;
+FUSE_LOG_ORG     = 0x50003C00;
```

Comment:
Can you also make these changes in `fmc/memory.x` and `runtime/memory.x`?

Reply:
Sure will do.

Reply:
Created a separate issue.

https://github.com/chipsalliance/caliptra-sw/issues/322

Resolving this comment.